install/gcp/bootstrap/gcp_envoy_bootstrap.json

```diff
@@ -0,0 +1,142 @@
+{
+  "node": {
+    "id": "{{ .nodeID }}",
+    "cluster": "{{ .cluster }}",
+    "locality": {
+      {{ if .region }}
+      "region": "{{ .region }}",
+      {{ end }}
+      {{ if .zone }}
+      "zone": "{{ .zone }}",
+      {{ end }}
+      {{ if .sub_zone }}
+      "sub_zone": "{{ .sub_zone }}",
+      {{ end }}
+    },
+    "metadata": {{ .meta_json_str }}
+  },
+  "static_resources": {
+    "listeners": [
+      {
+        "name": "virtual",
+        "address": {
+          "socket_address": {
+            "address": "0.0.0.0",
+            "port_value": 15001
+          }
+        },
+        "use_original_dst": true,
+        "filter_chains": [
+          {
+            "filters": [
```

Comment:
why http ? its safer to have TCP proxy here.

Reply:
I believe this is similar to default Istio template? Nevertheless, hold for a bit. I might send different PR (see comment from @costinm above).